great-idea/css/index.css

```diff
@@ -63,4 +58,177 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+}
+
+.navigation {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	width: 876px;
+	margin-top: 25px;
+}
+
+.navigation a {
+	text-decoration: none;
+	color: grey;
+}
+
+header {
+	height: 450px;
+	text-align: center;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-top: 25px;
+}
+
+.h1-div {
+	font-size: 75px;
+	width: 290px;
+}
+
+button {
+	text-decoration: none;
+	border: 1px solid black;
+	background: white;
+	font-size: 15px;
+	width: 140px;
+	height: 30px;
+}
+
+.top-content {
+	border-top: 2px solid black;
+	text-align: justify;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.top-content div {
+	width: 400px;
+}
+
+.bottom-content {
+	text-align: left;
+	border-bottom: 2px solid black;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.bottom-content div {
+	width: 280px;
+}
+
+.contact {
+	margin-top: 20px;
+	width: 140px;
+	height: 140px;
+}
+
+address {
+	line-height: 22px;
+}
+
+footer {
+	margin-top: 20px;
+	text-align: center;
+}
+
+/* services.html */
+
+.services-header-img {
+	padding-top: 30px;
+	padding-bottom: 30px;
+}
+
+.intro {
+	padding-bottom: 30px;
+	border-bottom: 2px solid black;
+}
+
+.main-content {
+	border-bottom: 2px solid black;
+	padding-top: 30px;
+	padding-bottom: 15px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+
+}
+
+.main-content div {
+	margin-bottom: 30px;
+	padding: 20px;
+	width: 420px;
+	height: 220px;
+	background: rgb(233, 230, 230);
+	border: 1px solid black;
+}
+
+.main-content button {
+	margin-top: 15px;
+	border-radius: 5px;
+}
+
+.services-bottom-section {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 30px;
+	margin-bottom: 60px;
+}
+
+.facts-header-box {
+	width: 400px;
+}
+
+
+ul {
+	margin-top: 10px;
+	padding-left: 18px;
+ 	list-style-type: square;
+}
+
+@media only screen and (max-width: 768px) {
+	body {
+		width: 100%;
+	}
+
```

Comment:
you no longer have to specify the type for your media query, you're now able to just say @media and then go right into your properties ex. @media (max-width: 768px)